assembl/static2/js/app/components/common/ChatFrame.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+import { connect } from 'react-redux';
+
+const varPath = (root, path) => {
+  return path.split('.').reduce((value, nextKey) => {
+    return value && value[nextKey];
+  }, root);
+};
+
+class DumbChatFrame extends React.Component {
+  static Modal = ({ src }) => {
+    return (
+      <div className="chatframe-modal">
+        <div className="chatframe-modal-header">
+          <span className="chatframe-icon chatframe-modal-icon assembl-icon-robot" />
+          <Translate value="chatframe.title" />
+        </div>
+        <iframe className="chatframe-modal-iframe" title="chatframe" src={src} />
+      </div>
+    );
+  };
```

Comment:
Why do use static component? Can't you just write them on the module scope?

Reply:
it's the chatframe modal so I figured it would be best declared in the chatframe class

Reply:
you prefer it that way?

Reply:
I'd prefer in the module scope yes, like we do in other modules. In some other modules, we define several non exported components on the module scope. I don't know why it should be different here. Let's use the same style everywhere.